NGitLab/Models/RunnerUpdate.cs

```diff
@@ -1,15 +1,20 @@
+using System;
 using System.Text.Json.Serialization;
 
 namespace NGitLab.Models;
 
 public class RunnerUpdate
 {
+    [Obsolete("Use Paused attribute instead. Will soon be removed.")]
```

Comment:
```suggestion
    [Obsolete("Use Paused field instead")]
```

There is no date planned atm and since it will be removed in the v5 of the API apparently, that will likely take time :).